airflow/models/baseoperator.py

```diff
@@ -502,7 +502,7 @@ def __init__(
         pool: Optional[str] = None,
         pool_slots: int = 1,
         sla: Optional[timedelta] = None,
-        execution_timeout: Optional[timedelta] = None,
+        execution_timeout: Optional[timedelta] = conf.gettimedelta('core', 'default_ execution_timeout', fallback=None),    
```

Comment:
there is no `gettimedelta` function in https://github.com/apache/airflow/blob/main/airflow/configuration.py . You need to implement it.

Reply:
execution_timeout accept timedelta object. You are reading value from a configuration file you need to convert it into the object.